src/containers/pages/OrganizationViews/SingleOrganizationView/index.tsx

```diff
@@ -0,0 +1,312 @@
+import ListView from '@onaio/list-view';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { capitalize, values } from 'lodash';
+import React, { useEffect, useState } from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { RouteComponentProps } from 'react-router';
+import AsyncSelect from 'react-select/async';
+import { ValueType } from 'react-select/src/types';
+import { Col, Row } from 'reactstrap';
+import { Store } from 'redux';
+import LinkAsButton from '../../../../components/LinkAsButton';
+import HeaderBreadcrumb, {
+  BreadCrumbProps,
+} from '../../../../components/page/HeaderBreadcrumb/HeaderBreadcrumb';
+import Loading from '../../../../components/page/Loading';
+import {
+  ACTIONS,
+  DETAILS,
+  EDIT_TEAM,
+  EDIT_TEAM_URL,
+  HOME,
+  HOME_URL,
+  IDENTIFIER,
+  MEMBERS,
+  NAME,
+  OPENSRP_ORGANIZATION_ENDPOINT,
+  ORGANIZATIONS_LABEL,
+  SINGLE_TEAM_URL,
+  TEAM_LIST_URL,
+  TEAMS,
+  USERNAME,
+} from '../../../../constants';
+import { OPENSRP_PRACTITIONER_ENDPOINT } from '../../../../constants';
+import PractitionerSelect from '../../../../containers/forms/PractitionerSelect';
+import { generateNameSpacedUUID, RouteParams } from '../../../../helpers/utils';
+import { OpenSRPService } from '../../../../services/opensrp';
+import store from '../../../../store';
+import organizationsReducer, {
+  fetchOrganizations,
+  getOrganizationById,
+  Organization,
+  reducerName as organizationsReducerName,
+} from '../../../../store/ducks/organizations';
+import * as fixtures from '../../../../store/ducks/tests/fixtures';
+
+reducerRegistry.register(organizationsReducerName, organizationsReducer);
+
+/** Placeholder interface for a organizationMember object schema */
+interface Practitioner {
+  identifier: string;
+  name: string;
+  username: string;
+}
+
+/** interface to describe props for SingleOrganizationView */
+interface SingleOrganizationViewProps {
+  organization: Organization | null;
+  practitioners: Practitioner[];
+  serviceClass: typeof OpenSRPService;
+  fetchOrganizationsAction: typeof fetchOrganizations;
+}
+
+/** the default props for SingleOrganizationView */
+const defaultProps: SingleOrganizationViewProps = {
+  fetchOrganizationsAction: fetchOrganizations,
+  organization: null,
+  practitioners: [],
+  serviceClass: OpenSRPService,
+};
+
+/** the interface for all SingleOrganizationView props  */
+type SingleOrgViewPropsType = SingleOrganizationViewProps & RouteComponentProps<RouteParams>;
+
+const SingleOrganizationView = (props: SingleOrgViewPropsType) => {
+  const { organization, practitioners, serviceClass, fetchOrganizationsAction } = props;
+
+  // props //
+
+  // props for the header breadcrumb
+  const basePage = {
+    label: TEAMS,
+    url: TEAM_LIST_URL,
+  };
+  const breadcrumbProps: BreadCrumbProps = {
+    currentPage: {
+      label: organization!.identifier,
+      url: `${SINGLE_TEAM_URL}/${organization!.identifier}`,
```

Comment:
This syntax does not seem to play well in tests, when presented with `null` this line (and other instances of this syntax) throw a type error by trying to reference `null.identifier `.

<img width="923" alt="Screen Shot 2019-10-03 at 9 33 15 PM" src="https://user-images.githubusercontent.com/4141527/66181439-a0f37900-e625-11e9-9c45-c45e716811d8.png">